doc/05-Authentication.md

```diff
@@ -158,6 +158,95 @@ resource = icingaweb-mysql
 Please read [this chapter](20-Advanced-Topics.md#advanced-topics-authentication-tips-manual-user-database-auth)
 in order to manually create users directly inside the database.
 
+### Password Policy <a id="authentication-password-policy"></a>
```

Comment:
Please add newlines after headings and speak of `Icinga Web` not `Icinga Web 2`.